### DISCUSSION TITLE
Canvas renders in Gatsby Develop, but not when deployed

### DISCUSSION THREAD

**Discussion by hamdERGler** [jeffreylanters/react-unity-webgl#193] [Help Wanted!]
Hi! Relatively new to webdev, and I hit a snag when I deployed the site that worked well in the dev environment to live. The canvas is just blank, but it is there? www.ergnomes.io/UnityPage

Any pointers on where to look are appreciated. Thanks!

running the latest of all packages.

**Comment by jeffreylanters** [comment]
Hi! Welcome to the world of web development. I took a look at your website and noticed an error in the console, noticing about the Unity builds not being found. This means your Unity build is not served right on your web server. I've never used Gatsby, but I understand the concept. So you have to try to make your Unity build reachable from your webserver for this to work.

Best of luck, keep me posted!
